mig/install/generateconfs.py

```diff
@@ -358,7 +359,18 @@ def usage(options):
         # Remove default values to use generate_confs default values
         if val == 'DEFAULT':
             del settings[key]
-    options = generate_confs(**settings)
+
+    if os.getenv('MIG_ENV', 'default') == 'local':
+        output_path = os.path.join(MIG_BASE, 'envhelp/output')
+    else:
+        output_path = os.getcwd()
+        # set user and group to hard-coded bakcwards compatible values
```

Comment:
I think this 'reverts' my intended change in PR75 to **always** use the current user/group if user arg is left to `AUTO`. I'd prefer to just leave it out as we always specify the user and/or generate as that user anyway.